src/features/emails/components/EmailEditor/EmailSettings/SettingsTab.tsx

```diff
@@ -21,6 +22,10 @@ const SettingsTab: FC<SettingsTabProps> = ({
   const { subject, emailAddress, setSubject, orgTitle } =
     useEmailSettings(initialSubject);
 
+  const debouncedFinishedTyping = useDebounce(async (value: string) => {
```

Comment:
I think this would make sense for the editor as a whole, e.g. when editing the content of the email.